src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+  const resualt = {};
+
+  for (let i = 0; i < keys.length; i++) {
+    resualt[keys[i]] = values[i];
```

Comment:
check here if result already have key that you're trying to add - return null, else - add reversed key